webapp/pages/post/_id/_slug/more-info.vue

```diff
@@ -1,8 +1,6 @@
 <template>
   <ds-card>
-    <h2 style="margin-bottom: .2em;">
-      Mehr Informationen
-    </h2>
+    <h2 style="margin-bottom: .2em;">Mehr Informationen</h2>
```

Comment:
You probably don't want to make more changes to push up, but it would be nice to have this translated.. I guess there's another ticket to work through the untranslated texts, right?

Reply:
I have made a screenshot to my list … @mattwr18